staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+
+### Compile the Contract
+1. Click on the Compile Tab, second from top
+2. Compile [Staking Interface.sol](https://github.com/PureStake/moonbeam/blob/master/runtime/precompiles/src/StakingInterface.sol)
+
+![Compiling StakingInteface.sol](/images/staking/compile4.png)
+
+### Access the Contract
+1. Click on the Deploy and Run Tab, directly below the Compile Tab in Remix
+2. Make sure "Injected Web3" is selected in the Environment drop down
+3. Make sure “ParachainStaking - StakingInterface.sol” is selected in the Contract dropdown. Since this is a precompiled contract on Moonbase Alpha there is no need to deploy, instead we are going to provide the address of the precompile in the “AtAddress” Field
+4. Provide the address of the Staking precompile: `{{networks.moonbase.staking.precompile_address}}` and Click “At Address”
+
+![Provide the addressl](/images/staking/deploy1export.png)
+
+### Nominate a Collator
+For this example, we are going to be nominating a collator. Nominators are token holders who stake tokens, vouching for specific collators. Any user that holds a minimum amount of {{networks.moonbase.staking.min_nom_stake}} tokens as free balance can become a nominator. 
+
+1. Expand the panel with the contract address and find the nominate function and expand the panel to see the parameters.
+2. Provide the address of a collator such as `{{ networks.moonbase.staking.collators.address1 }}`
+3. Provide the amount to nominate in WEI. There is a minimum of {{networks.moonbase.staking.min_nom_stake}} DEV to nominate, so the lowest amount in WEI is `5000000000000000000`
+4. Press Transact and Confirm the Transaction in Metamask
+
+![Nominate a Collator](/images/staking/nominateexport.png)
+
+### Verify Nomination
+To verify your nomination was successful, you can check the chain state in Polkadot.js Apps. First, add your metamask address to the [address book in Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/addresses). If you've already completed this step you can skip ahead to the next section. 
+
+#### Add Metamask Address to Address Book
+1. Click on Add Contact
+2. Add your Metamask Moonbase Alpha Address
+3. Provide a nickname for the account
+
+![Add to Address Book](/images/staking/addressbook.png)
+
+#### Verify Nominator State
+1. To verify your nomination was successful, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/chainstate) and navigate to Developer -> Chain State
+2. Select the parachainStaking pallet
+3. Select the nominatorState query
+4. Click the Plus Button to return the results and verify your nomination
+
+![Verify Nomination](/images/staking/verifynomination.png)
+
+### Revoking a Nomination
+To revoke a nomination and receive your tokens back, call the revoke_nomination method, providing the same address you started the nomination with above. You can check your Nominator state again on Polkadot.js Apps to confirm.
```

Comment:
maybe back ticks or quotes around `revoke_nomination` 🤷‍♀️ 

and does "Nominator" need to be capitalized?